subprojects/maven/src/main/java/org/gradle/api/publish/maven/internal/tasks/MavenPomFileGenerator.java

```diff
@@ -268,6 +272,9 @@ private void addDependencyManagement(MavenDependency dependency, String scope) {
         mavenDependency.setArtifactId(dependency.getArtifactId());
         mavenDependency.setVersion(mapToMavenSyntax(dependency.getVersion()));
         mavenDependency.setScope(scope);
+        if (scope.equals("import")) {
+            mavenDependency.setType("pom");
```

Comment:
It feels like `type` should be a nullable property on `org.gradle.api.publish.maven.MavenDependency`, rather than having this hard-coded mapping. This could assist us later to fix issues like #3170.